pkg/kopia/command/log_output_utils.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 The Kanister Authors.
```

Comment:
Looks like these are only parsing output from snapshot or manifest commands. Does it warrant a file name that describes the same?